### DISCUSSION TITLE
Feature: Scroll to switch tabs

### DISCUSSION THREAD

**Discussion by DuBistKomisch** [Able-Bear-Software/tab-nodes-tree#11] [Ideas / Feature Request]
This is a nice feature from Tree Style Tabs I'd love to see in Tab Nodes Tree as well, see it here: https://addons.mozilla.org/en-US/firefox/addon/tree-style-tab-mouse-wheel/

**Comment by zeroarst** [comment]
Interesting! I'm a fan of TST but hadn't tried this addon before. I just installed it to check it out, and it looks like it scrolls through tabs that aren't 'discarded,' which makes sense. Since we don't want to activate tabs that just scroll over and already put into sleep while scrolling, which could waste device resources....  If that's the case, implementing something similar shouldn't be too difficult. However, I personally think disregarded tabs shouldn't be skipped, so I'll need to consider how to approach this properly.

**Reply by DuBistKomisch** [reply]
I believe that behaviour is actually an option in that addon: "Skip collapsed tabs", visible in the store page screenshot. That's always been how I've used it too personally. Might be good to also have the option here if you end up implementing it. Thanks for checking it out!
